instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/util.py

```diff
@@ -20,15 +20,16 @@
 def _extract_conn_attributes(conn_kwargs):
     """ Transform redis conn info into dict """
     attributes = {
-        "db.type": "redis",
-        "db.instance": conn_kwargs.get("db", 0),
+        "db.system": "redis",
+        "db.name": conn_kwargs.get("db", 0),
```

Comment:
Actually, the footnote on the [OTel Database Specificatiosn](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#call-level-attributes-for-specific-technologies) seems says it is required to use `db.redis.database_index` if the connection is not `0`?

Reply:
Good catch, maybe create a new issue to address this? @lonewolf3739

Reply:
Sure, will do that.